inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -133,12 +135,22 @@ def arxiv_plot_extract(obj, eng):
             current_app.logger.exception(err)
             return
 
+        lb = LiteratureBuilder(source=obj.data['acquisition_source']['source'], record=obj.data)
         for idx, plot in enumerate(plots):
+            key = '{uuid}_{name}'.format(uuid=uuid.uuid4(), name=plot.get('name'))
             with open(plot.get('url')) as plot_file:
-                obj.files[plot.get('name')] = plot_file
-            obj.files[plot.get('name')]['description'] = u'{0:05d} {1}'.format(
+                obj.files[key] = plot_file
+            obj.files[key]['description'] = u'{0:05d} {1}'.format(
```

Comment:
there is no description key in [`_files`](http://inspire-schemas.readthedocs.io/en/latest/schemas/records/elements/records-files.html#records-files-json). You can remove this line.